Source/FullScan/Events.lua

```diff
@@ -0,0 +1,17 @@
+Auctionator.Events.Register("AUCTION_HOUSE_SHOW", function()
```

Comment:
I *really* want these functions defined elsewhere. I do not want this file becoming hundreds of lines long; one of the initial reasons I refactored these into separate modules was to make them easy to ready and find.

Reply:
The Events.Register should be in the appropriate module, but yeah, there's no special reason to use anonymous functions to do it. Fixed